tests/backup/backup_test.go

```diff
@@ -7297,3 +7298,155 @@ var _ = Describe("{ScheduleBackupCreationAllNS}", func() {
+			log.InfoD("Register source and destination cluster for backup")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			err = CreateSourceAndDestClusters(orgID, "", "", ctx)
+			log.FailOnError(err, "Creating Source and destination cluster")
+			clusterStatus, clusterUid = Inst().Backup.RegisterBackupCluster(orgID, SourceClusterName, "")
+			dash.VerifyFatal(clusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying backup cluster %s", SourceClusterName))
+		})
+		Step("Taking backup of applications", func() {
+			log.InfoD("Taking backup of applications")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for _, namespace := range bkpNamespaces {
+				backupName = fmt.Sprintf("%s-%s-%v", BackupNamePrefix, namespace, time.Now().Unix())
+				backupNamespaceMap[namespace] = backupName
+				err = CreateBackup(backupName, SourceClusterName, backupLocationName, backupLocationUID, []string{namespace}, labelSelectors, orgID, clusterUid, "", "", "", "", ctx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying backup creation %s", backupName))
+				backupNames = append(backupNames, backupName)
+			}
+		})
+		Step("Restoring the backed up application", func() {
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			log.InfoD("Restoring backed up applications")
+			for _, namespace := range bkpNamespaces {
+				backupName := backupNamespaceMap[namespace]
+				restoreName = fmt.Sprintf("%s-%s-%v", restoreNamePrefix, backupName, time.Now().Unix())
+				restoreNames = append(restoreNames, restoreName)
+				restoredNameSpace := fmt.Sprintf("%s-%s", namespace, "restored")
+				namespaceMapping[namespace] = restoredNameSpace
+				deploymentName, err = CreateCustomRestoreWithPVCs(restoreName, backupName, namespaceMapping, SourceClusterName, orgID, ctx, make(map[string]string), namespace)
+				deploymentBackupMap[backupName] = deploymentName
+				log.FailOnError(err, "Restoring of backup [%s] has failed with name [%s] in namespace [%s]", backupName, restoreName, restoredNameSpace)
+			}
+		})
+
+		Step("Validating restored resources", func() {
+			for _, namespace := range bkpNamespaces {
+				restoreNamespace := fmt.Sprintf("%s-%s", namespace, "restored")
+				backupName := backupNamespaceMap[namespace]
+				deploymentName = deploymentBackupMap[backupName]
+				deploymentStatus, err := apps.Instance().DescribeDeployment(deploymentName, restoreNamespace)
+				log.FailOnError(err, "unable to fetch deployment status for %v", deploymentName)
+				status := deploymentStatus.Conditions[1].Status
+				dash.VerifyFatal(status, v1.ConditionTrue, fmt.Sprintf("checking the deployment status for %v in namespace %v", deploymentName, restoreNamespace))
+				log.InfoD("Validating restored resources %v in namespace %s", deploymentName, restoreNamespace)
```

Comment:
log.InfoD not requires here. It is needed just after entering the step. VerifyFatal is taking care of sending the logs to dashboard anyway